cloud/app/web_components/robot-agent-device.jsx

```diff
@@ -467,22 +487,30 @@ const Device = (props) => {
           packageName="robot-agent"
         />
 
-      <Fold title="Configuration">
-        <div style={styles.row}>
-          {latestVersionData?.info?.config &&
-            <ConfigEditor info={latestVersionData.info}
-              updateConfig={
-                (modifier) => runCommand('updateConfig', {modifier}, log.debug)
-              }/>}
-        </div>
+      <Fold title="Configuration & status">
+          <>
```

Comment:
Can we use `<F>` instead of `<>`? That's what I use everywhere else, and is the reason for defining `F`. Some IDE features like auto-indent get thrown off by the empty tag.